e2e/integration/frontity-01/amp.spec.ts

```diff
@@ -0,0 +1,23 @@
+const assertTemplate = () => {
+  cy.get("html").should("have.attr", "amp");
+  cy.get("style[amp-boilerplate]").should("exist");
+  cy.get("style[amp-custom]").should("exist");
+  cy.get("style[amp-custom]").should("exist");
+  cy.get("body script").should("not.exist");
```

Comment:
This pretty much sums app the needed things for a functional AMP template. The only one that I'm not sure if should be part of this, is the amp script tag. Should we check for that as well?

Reply:
Yes, please :) 

We will also need the `<link rel="amphtml" href="https://domain.com/some-post/amp/" />` element like [Luis mentioned](https://community.frontity.org/t/amp-package/388/32?u=mmczaplinski)

Side note: the tests are absolutely fine, but I think that going forward we should intergate the [amphtml-validator](https://www.npmjs.com/package/amphtml-validator) into the e2e test flow.

I've gotten started on this in my PR to [Add the AMP Processors](https://github.com/frontity/frontity/pull/718) and now that we have the template, I think that I can finish with them once the parent (CSS in the head) PR is merged 🙂